CenterForOpenScience/ember-osf
# [IN-165] [Ember-OSF] Add banner component

## Purpose

Add banner component

## Summary of Changes

- Replace old donate-banner component with scheduled banner component.
- Update related files.

## Side Effects / Testing Notes
To be tested after this component has been added to OSF ember apps preprints, registries, reviews
- Create banners in admin app
- Make sure they show as created
- Inspect the banner with developer console and make sure `alt` fields are present
- Try different sizes to make sure the banner is responsive

## Ticket

https://openscience.atlassian.net/browse/IN-165

# Reviewer Checklist
